website/docs/advanced-concepts/embed-appsmith-into-existing-application.md

```diff
@@ -98,40 +98,43 @@ npx http-server
 
 Next, open [http://localhost:8080/dashboard.html](http://localhost:8080/dashboard.html) and it should open the HTML File.
 
-### Modifying Layout
+### Modifying layout
 
-If you want to get your app to use the whole page in your browser, you can still change your height and width parameters like so:
```

Comment:
@harshilp24 did you check if this is possible in the UI generated embed URL  before deleting it?

Reply:
yes, my bad,  they still need to implement the iframe using HTML code, will update this :(